### PR TITLE
new contribution

### PR DESCRIPTION
**Pull request by rusksr**
adding new stuff

### CODE REVIEW

**Comment by rusksr** [comment]
do you like this

**Comment by ruonkeys** [comment]
ya thats ok

File: feature-a.js [hunk 1]
```diff
@@ -1 +1,3 @@
-console.log("feature a added");
\ No newline at end of file
+console.log("feature a added");
+
+console.log("new stuff added by rusksr");
```
Comment (ruonkeys) [inline feature-a.js:3]: great work

**Review by ruonkeys — COMMENTED** [review]
thats it

File: feature-a.js (hunk 1 above)
Comment (rusksr) [reply feature-a.js:3]: thanks

**Pull request merged**